jdk/README.md

```diff
@@ -0,0 +1,162 @@
+## Using Red Hat Single Sign-On 7.5 container images with alternative JDK
+
+The Red Hat Single Sign-On 7.5 for OpenJDK container images use the [Red Hat OpenJDK 11](https://access.redhat.com/documentation/en-us/openjdk/11/html-single/getting_started_with_openjdk_11/index#openjdk-overview), a free and open source implementation of the Java Platform, Standard Edition (Java SE) by default.
+
+To provide an illustrative example on how to install an alternative JDK and instruct the Red Hat Single Sign-On 7.5 container images to use it, the 'ibm-semeru-open-11-jdk' subdirectory contains a definition of Red Hat Single Sign-On 7.5 container file switching the JDK runtime to the latest available release of [IBM Semeru Runtime Open Edition Java 11 (LTS)](https://github.com/ibmruntimes/semeru11-binaries/releases/latest).
+
+## Building the Red Hat Single Sign-On 7.5 container image with latest release of IBM Semeru 11 JDK Open Edition
+
+You can build this example by creating a new build using OpenShift CLI (oc) tool. Specify:
+* The vanilla `rh-sso-7/sso75-openshift-rhel8` image stream to use as the builder,
+* The Docker build strategy to use for build execution, and
+* The Git repository, branch name and particular context directory within the repository as the source of the build
+
+**Prerequisites**
```

Comment:
.Prerequisites

Reply:
Fixed in latest version. Thanks for pointing out!

Reply:
> .Prerequisites

@andymunro I will actually need to revert these changes back. The code isn't written in Asciidoc, but rather in Markdown syntax, so the dot before the string actually does nothing. See here, on how it's rendered:
* https://github.com/iankko/redhat-sso-7-openshift-image/tree/semeru/jdk

Instead of **Prerequisites**, it's just '.Prerequisites' (IOW without any effects).

The same case for the other remaining strings, that were modified. Are you OK with returning the previous form?

Thanks, Jan